bitwarden_license/src/Sso/appsettings.Development.json

```diff
@@ -11,7 +11,7 @@
       "internalAdmin": "http://localhost:62911",
       "internalIdentity": "http://localhost:33656",
       "internalApi": "http://localhost:4000",
-      "internalVault": "http://localhost:4001",
+      "internalVault": "https://localhost:8080",
```

Comment:
We should revert this change.

Reply:
I just saw your comment on the incorrect url. I'll need to investigate if it's used in other places and why it's pointing to 4001.

Reply:
I think we can keep this change.